src/main/java/de/tum/in/www1/artemis/domain/participation/Participation.java

```diff
@@ -85,6 +85,22 @@
     @Cache(usage = CacheConcurrencyStrategy.NONSTRICT_READ_WRITE)
     private Set<Submission> submissions = new HashSet<>();
 
+    /**
+     * This property stores the total number of submissions in this participation. Not stored in the database, computed dynamically and used in showing statistics to the user in
+     * the exercise view.
+     */
+    @Transient
+    @JsonProperty
+    private int submissionCount;
```

Comment:
This got refactored from `Result`. Should we also update `toString`?

Reply:
I added it to the `toString` method.

Reply:
Please use Integer, because this value might not be initialized

Reply:
Done